### PR TITLE
Features/gallery page and other

### PR DESCRIPTION
**Pull request by daria31v**

### CODE REVIEW

**Comment by daria31v** [comment]
Figma - https://www.figma.com/file/11l5GmPWI1cPKLPjQUAnwR/A-world-in-my-lens?type=design&node-id=0-1&mode=design&t=ffPXKIOwDsD14Ly6-0

File: public/images/beards.jpeg
Comment (pavelsuraba) [inline public/images/beards.jpeg:1]: Are you doing some image optimalization? Right image size and reducing image quality to to lower image size?

File: public/images/canon.jpeg
Comment (pavelsuraba) [inline public/images/canon.jpeg:1]: Remove this, this should be an png file and not jpeg. If it has a transparent background then it has to be a png.

File: public/images/fujifilm.png
Comment (pavelsuraba) [inline public/images/fujifilm.png:1]: Should be jpeg not png

File: public/images/window.jpeg
Comment (pavelsuraba) [inline public/images/window.jpeg:1]: Pay attention to the image size this is way too big.

File: src/app/components/Slider/gallerySlider.tsx [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+import sunrise from '../../../../public/images/photo_7.jpg'
+import sky from '../../../../public/images/sky.jpg'
+import kids from '../../../../public/images/photo_7.jpg'
+import lake from '../../../../public/images/photo_5.jpg'
+import sun from '../../../../public/images/sun.jpg'
+
+export const GallerySlider = () => {
+  const onInit = () => {
+    console.log('lightGallery has been initialized')
+  }
```
Comment (pavelsuraba) [inline src/app/components/Slider/gallerySlider.tsx]: I think you can remove this

File: tailwind.config.ts [hunk 2]
```diff
@@ -11,18 +11,20 @@ module.exports = {
       orange: '#F38605',
       duna: '#C25D05',
       white: '#fff',
-      'bg-white': '#F3F6FA'
+      blue_light: 'rgb(219 234 254);'
```
Comment (pavelsuraba) [inline tailwind.config.ts]: I would keep all color definitions in hex, you can convert it from rgb online if you need to.

**Comment by pavelsuraba** [comment]
Container is messed up again.
Screenshot 2023-11-21 at 19 38 03

File: public/images/beards.jpeg
Comment (daria31v) [reply public/images/beards.jpeg:1]: haven't done this yet

File: public/images/window.jpeg
Comment (daria31v) [reply public/images/window.jpeg:1]: this is a temporary solution
I'll keep it on mind

File: src/app/components/Slider/gallerySlider.tsx (hunk 1 above)
Comment (daria31v) [reply src/app/components/Slider/gallerySlider.tsx]: oh yes

File: tailwind.config.ts (hunk 2 above)
Comment (daria31v) [reply tailwind.config.ts]: I got

**Pull request merged**